superbench/runner/runner.py

```diff
@@ -211,6 +234,6 @@ def run(self):
                             'proc_rank': proc_rank
                         }) for proc_rank in range(mode.proc_num)
                     )
-                elif mode.name == 'torch.distributed':
+                elif mode.name == 'torch.distributed' or mode.name == 'mpi':
```

Comment:
Is our mode.name an enum variable? Here is no `else` to handle the failure scenario.

Reply:
added warning in else